recipes/pymarkovchain/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "PyMarkovChain" %}
+{% set version = "1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8a03fba5d9f390b528991f5e3411188b545aa16a7131cea44c5f9cf1814d08a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pymarkovchain
+
+about:
+  home: https://github.com/MaxWagner/PyMarkovChain
+  license: "BEER-WARE"
```

Comment:
I think the "BEER-WARE" license requires the `license_file`. I think it should be there in the source distribution, but I haven't checked.

Reply:
The license itself doesn't require much https://github.com/TehMillhouse/PyMarkovChain/blob/master/LICENSE.txt

Reply:
I interpretted "As long as you retain this notice..." as meaning you should include the notice with any distribution of the code.

Reply:
I see. So conda-forge requires the license file for any license, except may be unlicense. This should be clarified in the example recipe.